motr/io_req_fop.c

```diff
@@ -212,16 +225,14 @@ static void application_attribute_copy(struct m0_indexvec *rep_ivec,
 						      ti_goff_index,
 						      &ioo->ioo_ext,
 						      unit_size, cs_sz );
-
+		M0_ASSERT(dst);
```

Comment:
Please use tabs instead of spaces of alignment.

Reply:
done